packages/Alert/src/alert.scss

```diff
@@ -119,6 +119,7 @@
 
         &--inline,
         &--horizontal {
+            padding: 0.5rem;
```

Comment:
this needs to be a variable. it's probably different between igloo and workleap brand

Reply:
Totally, this was a placeholder.